tests/compute/switch-stmt.slang.1.expected.txt

```diff
@@ -0,0 +1,8 @@
+10
```

Comment:
What is this file for?

Reply:
If a test file has multiple `//TEST:...` lines, then we will use the same file to run multiple tests, where the "output stem" will take the form `name`, then `name.1`, then `name.2`, etc.

So this file is the expected output for the second `//TEST:...` line in `switch-stmt.slang`.